### PR TITLE
Fix creation of instance id for QA env due to echo -n different implementations

### PR DESCRIPTION
**Pull request by se7entyse7en**
Signed-off-by: Lou Marvin Caraig <loumarvincaraig@gmail.com>

### CODE REVIEW

**Comment by se7entyse7en** [comment]
@lwsanty as discussed in Slack, tested and it worked. 👍

File: Makefile.testenv [hunk 1]
```diff
@@ -3,7 +3,7 @@
 TEST_BRANCH ?=
 TEST_SERVICE_ACCOUNT ?=
 TEST_GCE_IMAGE ?= test-web-app
-TEST_GCE_INSTANCE ?= branch-$(shell sh -c "echo -n $(TEST_BRANCH) | md5sum | cut -c -32")
+TEST_GCE_INSTANCE ?= branch-$(shell sh -c "printf \"%s\" \"$(TEST_BRANCH)\" | md5sum | cut -c -32")
```
Comment (dpordomingo) [inline Makefile.testenv:6]: I wonder if this one can fail in macOS too https://github.com/athenianco/athenian-webapp/blob/3eb21c12d3a6a109bd8d9d2b3688b3e834baaca7/Makefile.testenv#L48

Comment (se7entyse7en) [reply Makefile.testenv:6]: Nope this is working fine as expected and printed in yellow 👍

**Comment by se7entyse7en** [comment]
@lwsanty I'm merging this since we discussed it in Slack and tested and working fine 👍

**Comment by se7entyse7en** [comment]
> Would it help to force the Makefile to use bash as done in athenianco/ci::Makefile.main

Sorry @dpordomingo I missed this. Anyway I wasn't aware that we're already doing it in main Makefile, then yes it makes sense.

**Comment by dpordomingo** [comment]
I'm not sure it will work because I remember that macos is also doing another weird thing with the shell interpreter, I don't remember but it was related to using another interpreter because it symlinks one to another.
(I remember we had that issue in our former company, of symlinking `sh` to `zsh` or something like that in macos)

**Pull request merged**